tests/test_series.py

```diff
@@ -1238,9 +1238,11 @@ def cond2(x: pd.Series[int]) -> pd.Series[bool]:
     check(assert_type(s.where(cond3, other=0), "pd.Series[int]"), pd.Series, int)
 
 
-def test_bitwise_operators_int() -> None:
+def test_bitwise_operators() -> None:
     s = pd.Series([1, 2, 3, 4])
-#     for issue #348 (bitwise operators on Series should support int)
+    # for issue #348 (bitwise operators on Series should support int)
+    check(assret_type(s & 3, "pd.Series[int]"), pd.Series,int)
+    check(assert_type(3 & s, "pd.Series[int]"), pd.Series,int)
```

Comment:
Please add the tests for the `|` and `^` operator as well, as you had in the other PR

Reply:
done sir